tests/fixtures/1_single_zone_tiny_house.json

```diff
@@ -0,0 +1,961 @@
+                    "name": "Generic Gypsum Board",
+                    "roughness": "MediumSmooth",
+                    "thickness": 0.0127,
+                    "conductivity": 0.16,
+                    "density": 800.0,
+                    "specific_heat": 1090.0,
+                    "thermal_absorptance": 0.9,
+                    "solar_absorptance": 0.5,
+                    "visible_absorptance": 0.5
+                },
+                {
+                    "type": "EnergyMaterial",
+                    "name": "Generic Acoustic Tile",
+                    "roughness": "MediumSmooth",
+                    "thickness": 0.02,
+                    "conductivity": 0.06,
+                    "density": 368.0,
+                    "specific_heat": 590.0,
+                    "thermal_absorptance": 0.9,
+                    "solar_absorptance": 0.3,
+                    "visible_absorptance": 0.3
+                },
+                {
+                    "type": "EnergyMaterial",
+                    "name": "Generic Metal Surface",
+                    "roughness": "Smooth",
+                    "thickness": 0.0015,
+                    "conductivity": 45.0,
+                    "density": 7690.0,
+                    "specific_heat": 410.0,
+                    "thermal_absorptance": 0.9,
+                    "solar_absorptance": 0.5,
+                    "visible_absorptance": 0.5
+                },
+                {
+                    "type": "EnergyMaterial",
+                    "name": "Thick Stone",
+                    "roughness": "Rough",
+                    "thickness": 0.3,
+                    "conductivity": 2.31,
+                    "density": 2322.0,
+                    "specific_heat": 832.0,
+                    "thermal_absorptance": 0.95,
+                    "solar_absorptance": 0.75,
+                    "visible_absorptance": 0.8
+                }
+            ]
+        }
+    },
+    "rooms": [
+        {
+            "type": "RoomAbridged",
+            "name": "TinyHouseZone",
+            "name_original": "Tiny House Zone",
+            "properties": {
+                "type": "RoomProperties",
+                "energy": {
+                    "type": "RoomEnergyPropertiesAbridged",
+                    "program_type": null,
+                    "construction_set": null,
+                    "people": null,
+                    "lighting": null,
+                    "electric_equipment": null,
+                    "gas_equipment": null,
+                    "infiltration": null,
+                    "ventilation": null
+                }
+            },
+            "faces": [
+                "TinyHouseZone_Bottom",
+                "TinyHouseZone_Front",
+                "TinyHouseZone_Right",
+                "TinyHouseZone_Back",
+                "TinyHouseZone_Left",
+                "TinyHouseZone_Top"
+            ],
+            "indoor_shades": [
+                "TinyHouseZone_Back_InShd0"
```

Comment:
Are indoor shades things like cubicle walls?  These would be InteriorPartitionSurfaces in OS.

Reply:
Essentially, yes. In this sample here they refer to the interior half of a light shelf but they are also intended to describe all types of interior furniture (including cubicle walls).  They will have more significance to Radiance simulations and detailed thermal comfort mapping than they do to the exported EnergyPlus model but, if you have a way to include them in the OpenStudio model, we should consider using it.  One thing we are sure we would like is to have the Room's `outdoor_shades` assigned as `Zone:Shading:Detailed` rather than `Site:Shading:Detailed` in the ultimate E+ model.  We can discuss some of the ways to make this happen on the call Wednesday.

Reply:
Cool, it would be good for us if we can differentiate between light shelves and interior partitions.  Is it possible to associate light shelves with a particular window?

Reply:
At the moment, the schema doesn't support shades for individual Apertures but @mostaphaRoudsari thinks that we will need this for Radiance and he [has opened an issue about it](https://github.com/ladybug-tools/honeybee-core/issues/11). We can discuss Wednesday but maybe for now we just focus on implementing Room-assigned shade and we can add Aperture-assigned shade to the schema afterwards.

Reply:
> it would be good for us if we can differentiate between light shelves and interior partitions.

Agreed!

> Is it possible to associate light shelves with a particular window?

I agree that it should be. As @chriswmackey pointed-out it is something that we are thinking to implement. Do you already have a property for this in OpenStudio model.

Reply:
We have an object `OS:DaylightingDevice:Shelf` that points to a `SubSurface`, `InteriorPartitionSurface` on the inside, and `ShadingSurface` on the outside.  It's similar to the `DaylightingDevice:Shelf` in E+.